src/triage/component/audition/regrets.py

```diff
@@ -218,6 +220,11 @@ def plot_all_selection_rules(
                      .format(regret_metric, regret_parameter))
         (plot_min, plot_max) = self.plot_bounds(regret_metric, regret_parameter)
 
+        if self.directory:
+            path_to_save = self.directory + f'/regret_distance_from_best_rules_{regret_metric}{regret_parameter}.png'
+        else:
+            path_to_save = self.directory
```

Comment:
I see how this works, but I think it would be more clear if in these else blocks we just set `path_to_save` specifically to None.

Reply:
Not sure @thcrock would like this any better, but a common, concise pattern is:

    path_to_save = self.directory and os.path.join(self.directory, …)

(depending on type-safety needs, and on resulting line length).

Reply:
:+1: